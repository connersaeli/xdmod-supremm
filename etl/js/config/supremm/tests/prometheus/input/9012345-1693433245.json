{
	"_id" : "8291026-1518721536",
	"summarization" : {
		"version" : "1.0.6",
		"complete" : true,
		"mergetime" : 0.0,
		"created" : 16934173010,
		"srcdir" : null,
		"elapsed" : 0.12180685997009277,
		"datasource": "prometheus"
	},
	"procDump" : {
		"unconstrained" : [],
		"constrained" : [
			"sh",
			"ssh",
			"/bin/bash",
			"/util/academic/R/R-3.4.1/lib64/R/bin/R.bin"
		],
		"cpusallowed" : {
			"node1" : {
				"node" : "cpn-d14-09",
				"cpu_list" : "0,1,2,3,4,5,6,7"
			},
			"node0" : {
				"node" : "cpn-d14-13",
				"cpu_list" : "0,1,2,3,4,5,6,7"
			}
		}
	},
	"process_memory" : {
		"error" : 5
	},
	"infiniband" : {
		"mlx4_0:1" : {
			"switch-out-packets" : {
				"med" : 35.5,
				"skw" : 0,
				"min" : 35,
				"max" : 36,
				"cov" : 0.019918500878494297,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 35.5
			},
			"switch-out-bytes" : {
				"med" : 10224,
				"skw" : 0,
				"min" : 10080,
				"max" : 10368,
				"cov" : 0.019918500878494297,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 10224
			},
			"switch-in-bytes" : {
				"med" : 10224,
				"skw" : 0,
				"min" : 10080,
				"max" : 10368,
				"cov" : 0.019918500878494297,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 10224
			},
			"switch-in-packets" : {
				"med" : 35.5,
				"skw" : 0,
				"min" : 35,
				"max" : 36,
				"cov" : 0.019918500878494297,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 35.5
			}
		}
	},
	"cpuperf" : {
		"flops" : {
			"std" : 30306921.994356133,
			"med" : 902562718.5,
			"skw" : -2.5491946398101732,
			"min" : 788287123,
			"max" : 916374963,
			"cov" : 0.03402449178502819,
			"cnt" : 16,
			"krt" : 6.388341353366547,
			"avg" : 890738418.25
		},
		"cpldref" : {
			"std" : 0.41295641238909525,
			"med" : 3.009701123804997,
			"skw" : 1.0727106269376487,
			"min" : 2.8260908677050405,
			"max" : 4.163791917770755,
			"cov" : 0.1286780833653582,
			"cnt" : 16,
			"krt" : -0.05930586196916421,
			"avg" : 3.2092210389595257
		},
		"cpiref" : {
			"std" : 0.11736394303879955,
			"med" : 0.909072107320728,
			"skw" : 1.1826209966074597,
			"min" : 0.84421096088459,
			"max" : 1.2281009744149582,
			"cov" : 0.12314717705164689,
			"cnt" : 16,
			"krt" : 0.17827034190223,
			"avg" : 0.9530380301740745
		}
	},
	"gpfs" : {
		"gpfs0" : {
			"write_bytes" : {
				"med" : 219668.5,
				"skw" : 0,
				"min" : 119156,
				"max" : 320181,
				"cov" : 0.6470938741240811,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 219668.5
			},
			"reads" : {
				"cnt" : 2,
				"avg" : 0
			},
			"writes" : {
				"med" : 19,
				"skw" : 0,
				"min" : 18,
				"max" : 20,
				"cov" : 0.0744322927564787,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 19
			},
			"read_bytes" : {
				"cnt" : 2,
				"avg" : 0
			}
		}
	},
	"cpu" : {
		"jobcpus" : {
			"all" : {
				"cnt" : 16
			},
			"softirq" : {
				"std" : 0.00034383640741380536,
				"med" : 0.0003509408790240777,
				"skw" : 1.1865148443683264,
				"min" : 0,
				"max" : 0.0012391573729863693,
				"cov" : 0.8180540604080333,
				"cnt" : 16,
				"krt" : 0.5566342646393836,
				"avg" : 0.0004203101287001801
			},
			"iowait" : {
				"std" : 0.011983064356676607,
				"med" : 0.01725536976210859,
				"skw" : 0.7585171305580878,
				"min" : 0.0031701303275801338,
				"max" : 0.04704016913319239,
				"cov" : 0.5797962288463865,
				"cnt" : 16,
				"krt" : -0.2400542897831075,
				"avg" : 0.02066771696759595
			},
			"system" : {
				"std" : 0.011414059692378953,
				"med" : 0.015161270371715366,
				"skw" : 0.84247833911956,
				"min" : 0.009640666082383873,
				"max" : 0.044255620463798905,
				"cov" : 0.5521467755344203,
				"cnt" : 16,
				"krt" : -0.7851650241802925,
				"avg" : 0.02067214769357539
			},
			"idle" : {
				"std" : 0.029141080932748302,
				"med" : 0.8277073920239215,
				"skw" : -0.5908583456323322,
				"min" : 0.7507449605609114,
				"max" : 0.8638605142655865,
				"cov" : 0.035634238348609015,
				"cnt" : 16,
				"krt" : -0.1136525172109426,
				"avg" : 0.8177831850273243
			},
			"user" : {
				"std" : 0.01591336929546012,
				"med" : 0.137756910840543,
				"skw" : 0.9188150560506502,
				"min" : 0.12134554420570623,
				"max" : 0.17651183172655566,
				"cov" : 0.11329737970913212,
				"cnt" : 16,
				"krt" : -0.09410076233820419,
				"avg" : 0.14045664018280427
			},
			"irq" : {
				"cnt" : 16,
				"avg" : 0
			},
			"nice" : {
				"cnt" : 16,
				"avg" : 0
			}
		},
		"nodecpus" : {
			"all" : {
				"cnt" : 16
			},
			"softirq" : {
				"std" : 0.00034383640741380536,
				"med" : 0.0003509408790240777,
				"skw" : 1.1865148443683264,
				"min" : 0,
				"max" : 0.0012391573729863693,
				"cov" : 0.8180540604080333,
				"cnt" : 16,
				"krt" : 0.5566342646393836,
				"avg" : 0.0004203101287001801
			},
			"iowait" : {
				"std" : 0.011983064356676607,
				"med" : 0.01725536976210859,
				"skw" : 0.7585171305580878,
				"min" : 0.0031701303275801338,
				"max" : 0.04704016913319239,
				"cov" : 0.5797962288463865,
				"cnt" : 16,
				"krt" : -0.2400542897831075,
				"avg" : 0.02066771696759595
			},
			"system" : {
				"std" : 0.011414059692378953,
				"med" : 0.015161270371715366,
				"skw" : 0.84247833911956,
				"min" : 0.009640666082383873,
				"max" : 0.044255620463798905,
				"cov" : 0.5521467755344203,
				"cnt" : 16,
				"krt" : -0.7851650241802925,
				"avg" : 0.02067214769357539
			},
			"idle" : {
				"std" : 0.029141080932748302,
				"med" : 0.8277073920239215,
				"skw" : -0.5908583456323322,
				"min" : 0.7507449605609114,
				"max" : 0.8638605142655865,
				"cov" : 0.035634238348609015,
				"cnt" : 16,
				"krt" : -0.1136525172109426,
				"avg" : 0.8177831850273243
			},
			"user" : {
				"std" : 0.01591336929546012,
				"med" : 0.137756910840543,
				"skw" : 0.9188150560506502,
				"min" : 0.12134554420570623,
				"max" : 0.17651183172655566,
				"cov" : 0.11329737970913212,
				"cnt" : 16,
				"krt" : -0.09410076233820419,
				"avg" : 0.14045664018280427
			},
			"irq" : {
				"cnt" : 16,
				"avg" : 0
			},
			"nice" : {
				"cnt" : 16,
				"avg" : 0
			}
		},
		"effcpus" : {
			"all" : 16,
			"softirq" : {
				"std" : 0.00034383640741380536,
				"med" : 0.0003509408790240777,
				"skw" : 1.1865148443683264,
				"min" : 0,
				"max" : 0.0012391573729863693,
				"cov" : 0.8180540604080333,
				"cnt" : 16,
				"krt" : 0.5566342646393836,
				"avg" : 0.0004203101287001801
			},
			"iowait" : {
				"std" : 0.011983064356676607,
				"med" : 0.01725536976210859,
				"skw" : 0.7585171305580878,
				"min" : 0.0031701303275801338,
				"max" : 0.04704016913319239,
				"cov" : 0.5797962288463865,
				"cnt" : 16,
				"krt" : -0.2400542897831075,
				"avg" : 0.02066771696759595
			},
			"system" : {
				"std" : 0.011414059692378953,
				"med" : 0.015161270371715366,
				"skw" : 0.84247833911956,
				"min" : 0.009640666082383873,
				"max" : 0.044255620463798905,
				"cov" : 0.5521467755344203,
				"cnt" : 16,
				"krt" : -0.7851650241802925,
				"avg" : 0.02067214769357539
			},
			"idle" : {
				"std" : 0.029141080932748302,
				"med" : 0.8277073920239215,
				"skw" : -0.5908583456323322,
				"min" : 0.7507449605609114,
				"max" : 0.8638605142655865,
				"cov" : 0.035634238348609015,
				"cnt" : 16,
				"krt" : -0.1136525172109426,
				"avg" : 0.8177831850273243
			},
			"user" : {
				"std" : 0.01591336929546012,
				"med" : 0.137756910840543,
				"skw" : 0.9188150560506502,
				"min" : 0.12134554420570623,
				"max" : 0.17651183172655566,
				"cov" : 0.11329737970913212,
				"cnt" : 16,
				"krt" : -0.09410076233820419,
				"avg" : 0.14045664018280427
			},
			"irq" : {
				"cnt" : 16,
				"avg" : 0
			},
			"nice" : {
				"cnt" : 16,
				"avg" : 0
			}
		}
	},
	"network" : {
		"ib0" : {
			"out-bytes" : {
				"cnt" : 2,
				"avg" : 0
			},
			"in-bytes" : {
				"cnt" : 2,
				"avg" : 0
			}
		},
		"lo" : {
			"out-bytes" : {
				"med" : 526075,
				"skw" : 0,
				"min" : 2680,
				"max" : 1049470,
				"cov" : 1.4070090908677775,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 526075
			},
			"in-bytes" : {
				"med" : 526075,
				"skw" : 0,
				"min" : 2680,
				"max" : 1049470,
				"cov" : 1.4070090908677775,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 526075
			}
		},
		"enp11s0f1" : {
			"out-bytes" : {
				"cnt" : 2,
				"avg" : 0
			},
			"in-bytes" : {
				"cnt" : 2,
				"avg" : 0
			}
		},
		"enp11s0f0" : {
			"out-bytes" : {
				"med" : 15853118,
				"skw" : 0,
				"min" : 15389827,
				"max" : 16316409,
				"cov" : 0.041328930720467325,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 15853118
			},
			"in-bytes" : {
				"med" : 57696053,
				"skw" : 0,
				"min" : 56313710,
				"max" : 59078396,
				"cov" : 0.033883222799513016,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 57696053
			}
		},
		"enp0s29f1u2" : {
			"out-bytes" : {
				"cnt" : 2,
				"avg" : 0
			},
			"in-bytes" : {
				"med" : 1495,
				"skw" : 0,
				"min" : 1430,
				"max" : 1560,
				"cov" : 0.06148754619013457,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 1495
			}
		}
	},
	"cores" : {
		"cnt" : 2,
		"avg" : 8
	},
	"nfs" : {
		"/srv" : {
			"read-normal" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-server" : {
				"cnt" : 2,
				"avg" : 0
			},
			"read-server" : {
				"cnt" : 2,
				"avg" : 0
			},
			"read-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-normal" : {
				"cnt" : 2,
				"avg" : 0
			}
		},
		"/user" : {
			"read-normal" : {
				"med" : 217120,
				"skw" : 0,
				"min" : 90160,
				"max" : 344080,
				"cov" : 0.8269553881673183,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 217120
			},
			"write-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-server" : {
				"med" : 4741,
				"skw" : 0,
				"min" : 0,
				"max" : 9482,
				"cov" : 1.414213562373095,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 4741
			},
			"read-server" : {
				"med" : 20263,
				"skw" : 0,
				"min" : 7474,
				"max" : 33052,
				"cov" : 0.892581416828185,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 20263
			},
			"read-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-normal" : {
				"med" : 1915,
				"skw" : 0,
				"min" : 0,
				"max" : 3830,
				"cov" : 1.414213562373095,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 1915
			}
		},
		"/projects" : {
			"read-normal" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-server" : {
				"cnt" : 2,
				"avg" : 0
			},
			"read-server" : {
				"cnt" : 2,
				"avg" : 0
			},
			"read-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-normal" : {
				"cnt" : 2,
				"avg" : 0
			}
		},
		"/util" : {
			"read-normal" : {
				"med" : 99326299.5,
				"skw" : 0,
				"min" : 91938520,
				"max" : 106714079,
				"cov" : 0.10518762923128858,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 99326299.5
			},
			"write-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-server" : {
				"cnt" : 2,
				"avg" : 0
			},
			"read-server" : {
				"med" : 36865837.5,
				"skw" : 0,
				"min" : 35903596,
				"max" : 37828079,
				"cov" : 0.03691262892313868,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 36865837.5
			},
			"read-direct" : {
				"cnt" : 2,
				"avg" : 0
			},
			"write-normal" : {
				"cnt" : 2,
				"avg" : 0
			}
		}
	},
	"uncperf" : {
		"membw" : {
			"med" : 24540746752,
			"skw" : 0,
			"min" : 23585545984,
			"max" : 25495947520,
			"cov" : 0.05504550837617463,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 24540746752
		}
	},
	"memory" : {
		"used_minus_cache" : {
			"med" : 341228.7,
			"skw" : 0,
			"min" : 330293.2,
			"max" : 352164.2,
			"cov" : 0.04532189822055114,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 341228.7
		},
		"used" : {
			"med" : 2293700.8499999996,
			"skw" : 6.342235459246879e-15,
			"min" : 2183447.4,
			"max" : 2403954.3,
			"cov" : 0.06797831735050536,
			"cnt" : 2,
			"krt" : -1.9999999999999998,
			"avg" : 2293700.8499999996
		}
	},
	"load1" : {
		"maxpercore" : {
			"med" : 0.1431249976158142,
			"skw" : 0,
			"min" : 0.10750000178813934,
			"max" : 0.17874999344348907,
			"cov" : 0.3520094539614928,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 0.1431249976158142
		},
		"max" : {
			"med" : 1.1449999809265137,
			"skw" : 0,
			"min" : 0.8600000143051147,
			"max" : 1.4299999475479126,
			"cov" : 0.3520094539614928,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 1.1449999809265137
		},
		"meanpercore" : {
			"med" : 0.04083333319673935,
			"skw" : 0,
			"min" : 0.03833333232129613,
			"max" : 0.043333334072182574,
			"cov" : 0.08658453442832667,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 0.04083333319673935
		},
		"mean" : {
			"med" : 0.3266666655739148,
			"skw" : 0,
			"min" : 0.30666665857036907,
			"max" : 0.3466666725774606,
			"cov" : 0.08658453442832667,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 0.3266666655739148
		}
	},
	"acct" : {
		"uid" : 1,
		"exit_status" : "COMPLETED",
		"resource_id" : 13,
		"ncpus" : 16,
		"host_list" : [
			"cpn-d14-09",
			"cpn-d14-13"
		],
		"id" : "9012345",
		"local_job_id" : 9012345,
		"group" : "groupname",
		"reqmem" : "2800Mc",
		"eligible" : 1518721453,
		"exit_code" : "0:0",
		"submit" : 1518721453,
		"gid" : 16038,
		"reqcpus" : 16,
		"nodes" : 2,
		"start_time" : 1518721476,
		"jobname" : "jobname",
		"user" : "username",
		"account" : "accountname",
		"partition" : "default",
		"timelimit" : 3600,
		"end_time" : 1518721536
	},
	"catastrophe" : {
		"value" : 0.7260626302138242
	},
	"block" : {
		"sda" : {
			"write" : {
				"med" : 282.5,
				"skw" : 0,
				"min" : 260,
				"max" : 305,
				"cov" : 0.11263647841909608,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 282.5
			},
			"read" : {
				"med" : 173.5,
				"skw" : 0,
				"min" : 158,
				"max" : 189,
				"cov" : 0.12634184562987305,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 173.5
			},
			"write_bytes" : {
				"med" : 4222.5,
				"skw" : 0,
				"min" : 3385,
				"max" : 5060,
				"cov" : 0.2804982494937755,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 4222.5
			},
			"read_bytes" : {
				"med" : 6654,
				"skw" : 0,
				"min" : 6536,
				"max" : 6772,
				"cov" : 0.02507923059212883,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 6654
			}
		},
		"sdb" : {
			"write" : {
				"med" : 3,
				"skw" : 0,
				"min" : 1,
				"max" : 5,
				"cov" : 0.9428090415820635,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 3
			},
			"read" : {
				"med" : 4.5,
				"skw" : 0,
				"min" : 0,
				"max" : 9,
				"cov" : 1.414213562373095,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 4.5
			},
			"write_bytes" : {
				"med" : 30.5,
				"skw" : 0,
				"min" : 8,
				"max" : 53,
				"cov" : 1.0432723001112996,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 30.5
			},
			"read_bytes" : {
				"med" : 144,
				"skw" : 0,
				"min" : 0,
				"max" : 288,
				"cov" : 1.414213562373095,
				"cnt" : 2,
				"krt" : -2,
				"avg" : 144
			}
		}
	},
	"nodememory" : {
		"maxused" : {
			"med" : 17799762,
			"skw" : 0,
			"min" : 16862996,
			"max" : 18736528,
			"cov" : 0.07442724132884443,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 17799762
		},
		"used" : {
			"med" : 17706792.799999997,
			"skw" : 6.342192316333682e-15,
			"min" : 16824763.2,
			"max" : 18588822.4,
			"cov" : 0.07044631045406008,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 17706792.799999997
		},
		"physmem" : {
			"med" : 24511458,
			"skw" : 0,
			"min" : 24511456,
			"max" : 24511460,
			"cov" : 1.153920392963238e-7,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 24511458
		},
		"free" : {
			"med" : 6804665.199999999,
			"skw" : 1.5855588648117167e-15,
			"min" : 5922637.6,
			"max" : 7686692.8,
			"cov" : 0.1833117953117505,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 6804665.199999999
		},
		"maxfree" : {
			"med" : 6909256,
			"skw" : 0,
			"min" : 6114312,
			"max" : 7704200,
			"cov" : 0.16271224949938426,
			"cnt" : 2,
			"krt" : -2,
			"avg" : 6909256
		}
	}
}
