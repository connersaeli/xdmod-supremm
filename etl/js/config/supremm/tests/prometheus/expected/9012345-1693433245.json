{
    "local_job_id": {
        "value": "9012345",
        "error": 0
    },
    "name": {
        "value": "jobname",
        "error": 0
    },
    "resource_name": {
        "value": "prometheus",
        "error": 0
    },
    "resource_id": {
        "value": 2,
        "error": 0
    },
    "organization_id": {
        "value": 1,
        "error": 0
    },
    "account": {
        "value": "accountname",
        "error": 0
    },
    "username": {
        "value": "username",
        "error": 0
    },
    "cwd": {
        "value": "NA",
        "error": 2
    },
    "executable": {
        "value": "/util/academic/R/R-3.4.1/lib64/R/bin/R.bin",
        "error": 0
    },
    "application": {
        "value": "r",
        "error": 0
    },
    "exit_status": {
        "value": "COMPLETED",
        "error": 0
    },
    "datasource": {
        "value": "prometheus",
        "error": 0
    },
    "granted_pe": {
        "value": 16,
        "error": 0
    },
    "queue": {
        "value": "default",
        "error": 0
    },
    "requested_nodes": {
        "value": 2,
        "error": 0
    },
    "hosts": {
        "value": [
            "cpn-d14-09",
            "cpn-d14-13"
        ],
        "error": 0
    },
    "nodes": {
        "value": 2,
        "error": 0
    },
    "shared": {
        "value": 0,
        "error": 0
    },
    "cores": {
        "value": 16,
        "error": 0
    },
    "cores_avail": {
        "value": 16,
        "error": 0
    },
    "submit_time_ts": {
        "value": 1518721453,
        "error": 0
    },
    "eligible_time_ts": {
        "value": 1518721453,
        "error": 0
    },
    "start_time_ts": {
        "value": 1518721476,
        "error": 0
    },
    "end_time_ts": {
        "value": 1518721536,
        "error": 0
    },
    "wall_time": {
        "value": 60,
        "error": 0
    },
    "requested_wall_time": {
        "value": 3600,
        "error": 0
    },
    "wait_time": {
        "value": 23,
        "error": 0
    },
    "cpu_time": {
        "value": 960,
        "error": 0
    },
    "node_time": {
        "value": 120,
        "error": 0
    },
    "cpu_idle": {
        "value": 0.8177831850273243,
        "error": 0
    },
    "cpu_system": {
        "value": 0.02067214769357539,
        "error": 0
    },
    "cpu_user": {
        "value": 0.14045664018280427,
        "error": 0
    },
    "error": {
        "value": null,
        "error": 2
    },
    "flops": {
        "value": 890738418.25,
        "error": 0
    },
    "flops_cov": {
        "value": 0.03402449178502819,
        "error": 0
    },
    "cpiref": {
        "value": 0.9530380301740745,
        "error": 0
    },
    "cpiref_cov": {
        "value": 0.12314717705164689,
        "error": 0
    },
    "catastrophe": {
        "value": 0.7260626302138242,
        "error": 0
    },
    "cpldref": {
        "value": 3.2092210389595257,
        "error": 0
    },
    "cpldref_cov": {
        "value": 0.1286780833653582,
        "error": 0
    },
    "mem_transferred": {
        "value": 24540746752,
        "error": 0
    },
    "mem_transferred_cov": {
        "value": 0.05504550837617463,
        "error": 0
    },
    "cpu_user_imbalance": {
        "value": 31.25359188743257,
        "error": 0
    },
    "cpu_user_cv": {
        "value": 0.11329737970913212,
        "error": 0
    },
    "node_cpu_idle": {
        "value": 0.8177831850273243,
        "error": 0
    },
    "energy": {
        "value": null,
        "error": 8
    },
    "max_power": {
        "value": null,
        "error": 8
    },
    "memory_used": {
        "value": 349418188.8,
        "error": 0
    },
    "memory_used_cov": {
        "value": 0.04532189822055114,
        "error": 0
    },
    "max_memory": {
        "value": null,
        "error": 5
    },
    "mem_used_including_os_caches": {
        "value": 2348749670.3999996,
        "error": 0
    },
    "mem_used_including_os_caches_cov": {
        "value": 0.06797831735050536,
        "error": 0
    },
    "ib_rx_bytes": {
        "value": 10224,
        "error": 0
    },
    "block_sda_wr_ios": {
        "value": 282.5,
        "error": 0
    },
    "block_sda_wr_bytes": {
        "value": 4222.5,
        "error": 0
    },
    "block_sda_wr_bytes_cov": {
        "value": 0.2804982494937755,
        "error": 0
    },
    "block_sda_rd_ios": {
        "value": 173.5,
        "error": 0
    },
    "block_sda_rd_bytes": {
        "value": 6654,
        "error": 0
    },
    "block_sda_rd_bytes_cov": {
        "value": 0.02507923059212883,
        "error": 0
    },
    "netdrv_gpfs_rx": {
        "value": 0,
        "error": 0
    },
    "netdrv_gpfs_rx_cov": {
        "value": 0,
        "error": 0
    },
    "netdrv_gpfs_rx_msgs": {
        "value": 0,
        "error": 0
    },
    "netdrv_gpfs_tx": {
        "value": 219668.5,
        "error": 0
    },
    "netdrv_gpfs_tx_cov": {
        "value": 0.6470938741240811,
        "error": 0
    },
    "netdrv_gpfs_tx_msgs": {
        "value": 19,
        "error": 0
    },
    "netdrv_isilon_rx": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_rx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_rx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx_msgs": {
        "value": null,
        "error": 2
    },
    "netdir_home_read": {
        "value": 237383,
        "error": 0
    },
    "netdir_home_write": {
        "value": 6656,
        "error": 0
    },
    "netdir_projects_read": {
        "value": 0,
        "error": 0
    },
    "netdir_projects_write": {
        "value": 0,
        "error": 0
    },
    "netdir_util_read": {
        "value": 136192137,
        "error": 0
    },
    "netdir_util_write": {
        "value": 0,
        "error": 0
    },
    "net_eth0_rx": {
        "value": null,
        "error": 8
    },
    "net_eth0_rx_cov": {
        "value": null,
        "error": 8
    },
    "net_eth0_rx_packets": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx_cov": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx_packets": {
        "value": null,
        "error": 8
    },
    "net_ib0_rx": {
        "value": 0,
        "error": 0
    },
    "net_ib0_rx_packets": {
        "value": null,
        "error": 8
    },
    "net_ib0_tx": {
        "value": 0,
        "error": 0
    },
    "net_ib0_tx_packets": {
        "value": null,
        "error": 8
    },
    "gpus": {
        "value": 0,
        "error": 8
    },
    "gpu_time": {
        "value": null,
        "error": 8
    },
    "gpu_usage": {
        "value": null,
        "error": 8
    },
    "gpu_energy": {
        "value": null,
        "error": 2
    },
    "gpu_max_power": {
        "value": null,
        "error": 2
    },
    "gpu0_nv_mem_used": {
        "value": null,
        "error": 8
    },
    "gpu0_nv_utilization": {
        "value": null,
        "error": 8
    }
}
